requirements/edx/base.txt

```diff
@@ -96,7 +96,7 @@ django-ses==0.8.4
 django-simple-history==2.7.0
 django-splash==0.2.2
 django-statici18n==1.4.0
-django-storages==1.4.1
+django-storages==1.5.2
```

Comment:
I know we've discussed bumping `boto3` to the earliest available version with encryption support, but do you think it's possible to bump the requirement to the latest version? 
It's ok if it's not possible.

Reply:
I have implemented the minimum change discussed beforehand to make it simpler, but I can certainly try to update django-storages to the latest version. I'll tell you how it goes.

Reply:
django-storages 1.9.1 as worked without requiring any code changes. I have updated the PR with it.